doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -13,4 +13,284 @@ Contributing Development
 * What are the requirements for a code to be accepted?
 
 * What platforms are supported
+
+================================
+ ufs-community/ufs-weather-model
+================================
+
```

Comment:
Done

Reply:
Valbona, thank you for making these changes. When you are ready, please update the files in your fork/branch. This will cause the PR to be updated so we can see the changes. Tks

Reply:
Ligia, I sent the final copy to Jun and Dusan for review and as soon I have their comments I will update the file in my fork.